.env

```diff
@@ -14,6 +14,94 @@ HOST_URL=${SERVER_SCHEME}://${HOST_NAME}
 TIMEZONE=UTC
 GITPOD_ENV=false
 ENABLE_SSO=false
+
+# Bahmni EMR
+
+RUN_WITH_BAHMNI_EMR=false
+BAHMNI_TEST_ORDER_TYPE_UUID=f8ae333e-9d1a-423e-a6a8-c3a0687ebcf2
+
+# Mail Config Properties Environment Variables
+MAIL_TRANSPORT_PROTOCOL=smtps
+MAIL_SMTP_AUTH=true
+MAIL_SMTP_STARTTLS_ENABLE=true
+MAIL_SMTP_SSL_ENABLE=true
+MAIL_DEBUG=false
+SEND_MAIL=false
+MAIL_FROM=
+MAIL_SMTP_HOST=
+MAIL_SMTP_PORT=
+MAIL_USER=
+MAIL_PASSWORD=
+
+# Bahmni Config Environment Variables
+BAHMNI_CONFIG_IMAGE_TAG=1.0.0
+BAHMNI_CONFIG_VOLUME=bahmni-config
+BAHMNI_CONFIG_OVERRIDE_PATH=
+
+# Bahmni - OpenMRS Environment Variables
+OPENMRS_HOST=openmrs
+OPENMRS_PORT=8080
+OPENMRS_IMAGE_TAG=1.1.1
+OPENMRS_DB_IMAGE_NAME=bahmni/openmrs-db:1.0.0-standard
+OPENMRS_DB_NAME=openmrs
+OPENMRS_DB_HOST=openmrsdb
+OPENMRS_DB_USERNAME=openmrs-user
+OPENMRS_DB_PASSWORD=password
+OPENMRS_DB_SQL_MODES='STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION'
+OPENMRS_DB_CREATE_TABLES='true'
+OPENMRS_DB_AUTO_UPDATE='true'
+OPENMRS_MODULE_WEB_ADMIN='false'
+# OMRS_DEV_DEBUG_PORT=
+OMRS_JAVA_SERVER_OPTS=''
+OMRS_JAVA_MEMORY_OPTS=''
+OPENMRS_UPLOAD_FILES_PATH=./openmrs-uploads
+BAHMNI_OPENMRS_MODULES_PATH=
+OPENMRS_DB_TYPE=mysql
+OMRS_C3P0_MAX_SIZE=50
+WEIGHT_CONCEPT_UUID=5089AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+HEIGHT_CONCEPT_UUID=5090AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+OPENMRS_DOCKER_ENV='true'
+OMRS_DB_EXTRA_ARGS="&zeroDateTimeBehavior=convertToNull"
+LUCENE_MATCH_TYPE="START"
+DOCUMENT_MAX_SIZE_MB=
+
+# Bahmni Web Environment Variables
+BAHMNI_WEB_IMAGE_TAG=1.1.0
+BAHMNI_APPS_PATH=
+
+# Implementer Interface Environment Variables
+IMPLEMENTER_INTERFACE_IMAGE_TAG=1.1.1
+IMPLEMENTER_INTERFACE_CODE_PATH=
+
+# Bahmni Reports Environment Variables
+REPORTS_DB_IMAGE_NAME=mysql:8.0
+REPORTS_DB_NAME=bahmni_reports
+REPORTS_DB_USERNAME=reports-user
+REPORTS_DB_PASSWORD=password
+REPORTS_IMAGE_TAG=1.1.0
+
+#Patient Documents environmental variables
+PATIENT_DOCUMENTS_TAG=1.1.1
+
+#Appointments environmental variables
+APPOINTMENTS_IMAGE_TAG=1.1.1
```

Comment:
Docker images and tags are not meant to be altered during deployment. I recommend keeping this configuration within the Docker Compose file and not exposing it as configurable.

Reply:
I guess it's okay to assume the above, however we kind of implement this kind of thing on the `O3_DOCKER_IMAGE_TAG` prop.